### PR TITLE
[feature] Create build info from multiple repositories

### PR DESCRIPTION
**Pull request by danimtb**
related to #91

### CODE REVIEW

File: extensions/commands/art/cmd_build_info.py [hunk 1]
```diff
@@ -106,15 +106,32 @@ def _get_requested_by(nodes, node_id, artifact_type):
...
+        api_request("get", request_url, artifactory_user, artifactory_password)
+        artifact_exists = True
+    except:
+        pass
+    return artifact_exists
+
+def _get_artifact_path_from_artifactory(artifactory_url, repositories, file_path, artifactory_user, artifactory_password):
+    for repository in repositories:
+        artifact_path = f'{repository}/{file_path}'
+        if _check_artifact_exists(artifact_path, artifactory_url, artifactory_user, artifactory_password)
```
Comment (brianwjfrog) [inline extensions/commands/art/cmd_build_info.py:122]: Line 122 need a colon(:) at the end.

Comment (danimtb) [reply extensions/commands/art/cmd_build_info.py:122]:
```suggestion
        if _check_artifact_exists(artifact_path, artifactory_url, artifactory_user, artifactory_password):
```

Comment (danimtb) [reply extensions/commands/art/cmd_build_info.py]: done! Thanks 😄

**Comment by czoido** [comment]
Closing this in favor of the solution in https://github.com/conan-io/conan-extensions/pull/95 that will not error if certain artifacts are not found in the remote repository.

**Pull request closed without merging**